sunpy/net/hek/utils.py

```diff
@@ -0,0 +1,340 @@
+import re
+import json
+import warnings
+
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import ICRS, SkyCoord
+from astropy.table import Column, MaskedColumn
+from astropy.utils.data import get_pkg_data_filename
```

Comment:
can we sack off using this and use `importlib.resources` instead?

Like [this](https://github.com/DKISTDC/dkist/blob/937303bc3f1a47a2d08358edc15ce92a8ead94c5/dkist/net/attrs_values.py#L58)

Reply:
🤷 I don't have an opinion here, but if we're going to do that we should do it everywhere in the package which I will claim is not in scope here.